layers/+source-control/git/README.org

```diff
@@ -142,7 +142,7 @@ Here are the often used bindings inside a =status buffer=:
 | ~b b~       | checkout a branch                                   |
 | ~b c~       | create a branch                                     |
 | ~f f~       | fetch changes                                       |
-| ~F -r F~    | pull and rebase                                     |
+| ~F~         | Open a =pull buffer=                                |
```

Comment:
this line is about `pull and rebase` not about `open a pull buffer`. According to new Magit it is now `F r u` but this is a bit more tricky since the `r` value is now sticky. I would go for `F r u` and update the description to `pull tracked branch and rebase`.